#14214. Delete class CaptureTrackSource and move the code into VideoManager (#762)
meganz/MEGAchat: WebrtcAndroid.md

  `cp -R media <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R rtc_base <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R stats <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R video <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R audio <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R call <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R crypto <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R modules <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R pc <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp -R sdk <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`   
  `cp -R system_wrappers <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
  `cp common_types.h <Android_Path>/android2/app/src/main/jni/megachat/webrtc/include/webrtc/`  
 

>  If you have some doubt about Android project, you can review https://github.com/meganz/android2

Reviewer: ```suggestion
Should you have any question about the Android project, you can check https://github.com/meganz/android2.
```
